stable/atlantis/README.md

```diff
@@ -1,72 +1,71 @@
+| `github.hostname`                           | Hostname of your GitHub Enterprise installation.                                                                                                                                                                                                                                                          | n/a     |
+| `gitlab.user`                               | Repository or organization-wide secret for the Atlantis GitLab,integration. All repositories in GitLab that are to be integrated with Atlantis must share the same value.                                                                                                                                 | n/a     |
+| `gitlab.token`                              | Personal access token for the Atlantis GitLab user.                                                                                                                                                                                                                                                       | n/a     |
+| `gitlab.secret`                             | Webhook secret for the Atlantis GitLab integration. All repositories in GitLab that are to be integrated with Atlantis must share the same value.                                                                                                                                                         | n/a     |
+| `gitlab.hostname`                           | Hostname of your GitLab Enterprise installation.                                                                                                                                                                                                                                                          | n/a     |
+| `podTemplate.annotations`                   | Additional annotations to use for the StatefulSet.                                                                                                                                                                                                                                                        | n/a     |
+| `logLevel`                                  | Level to use for logging. Either debug, info, warn, or error.                                                                                                                                                                                                                                             | n/a     |
+| `orgWhiteList`                              | Whitelist of repositories from which Atlantis will accept webhooks. **This value must be set for Atlantis to function correctly.** Accepts wildcard characters (`*`). Multiple values may be comma-separated.                                                                                             | none    |
```

Comment:
it regrettably seems this value was [copy-pasted](https://github.com/helm/charts/pull/12506/files#diff-d1fed5f153a80b41fb4038e8b89405f6L54) but does not match [`orgWhitelist` in `values.yaml`](https://github.com/helm/charts/blob/master/stable/atlantis/values.yaml#L10). I do see it is correctly cased [above](https://github.com/helm/charts/pull/12506/files#diff-d1fed5f153a80b41fb4038e8b89405f6R20) so reasonable people can differ about how big of a deal it is, but FYI

Reply:
@mdaniel good catch, would love a PR to fix it!